tools/deploy_docs.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+if [[ $TRAVIS_PULL_REQUEST == false && $TRAVIS_BRANCH == "master" ]]
+then
+    echo "-- pushing docs --"
+
+    (
+    git config user.email "travis@travis-ci.com"
+    git config user.name "Travis Bot"
+
+    git clone --quiet --branch=gh-pages https://${GH_REF} doc_build
```

Comment:
I tested this sequence of commands on my machine, but we won't know for sure if it works until someone hits the big green button.  :smile: